src/EnergyPlus/ChillerElectricEIR.cc

```diff
@@ -1346,14 +1346,14 @@ void ElectricEIRChillerSpecs::initialize(EnergyPlusData &state, bool const RunFl
             state.dataLoopNodes->Node(state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).TempSetPointNodeNum).TempSetPointHi;
     }
 
-    Real64 mdot = 0.0;
+    this->EvapMassFlowRate = 0.0;
     Real64 mdotCond = 0.0;
-    if ((std::abs(MyLoad) > 0.0) && RunFlag) {
-        mdot = this->EvapMassFlowRateMax;
+    if ((MyLoad < 0.0) && RunFlag) {
+        this->EvapMassFlowRate = this->EvapMassFlowRateMax;
         mdotCond = this->CondMassFlowRateMax;
     }
 
-    PlantUtilities::SetComponentFlowRate(state, mdot, this->EvapInletNodeNum, this->EvapOutletNodeNum, this->CWPlantLoc);
+    PlantUtilities::SetComponentFlowRate(state, this->EvapMassFlowRate, this->EvapInletNodeNum, this->EvapOutletNodeNum, this->CWPlantLoc);
```

Comment:
Tried to set `EvapMassFlowRate` up in `initialize` but that starts introducing diffs. The chiller inlet node temp is slightly different from before in the first timestep when the chiller comes on. Note that just getting rid of `abs` here did not introduce any diffs. I'm running just the three icestorage files and getting diffs just in IceStorage-Parallel. CI might show other files with diffs.

Reply:
You didn't change anything else (and the abs isn't the cause). So why would using a report variable to make this initialization be any different than what was here originally? I wonder if it's this in update that is getting hit now?

        if (this->CondMassFlowRate < DataBranchAirLoopPlant::MassFlowTolerance &&
            this->EvapMassFlowRate < DataBranchAirLoopPlant::MassFlowTolerance) {